ntbs-service/Models/Entities/Notification.cs

```diff
@@ -130,6 +128,7 @@ public Notification()
         public string FormattedDob => PatientDetails.Dob.ConvertToString();
         [Display(Name = "Date created")]
         public string FormattedCreationDate => CreationDate.ConvertToString();
+        [Display(Name = "Date notified")]
```

Comment:
We had been using submission date as the notification date for ages on the home page, thought that must be a residual mistake from before we had notification date?